README.md

```diff
@@ -1,9 +1,67 @@
-# XCP-ng
+# XCP-ng: A turnkey open source virtualization platform
 
-XCP-ng is a "reboot" of [old XCP](https://wiki.xen.org/wiki/XCP_Overview), which was ditched when XenServer was Open Sourced. Since XenServer 7.3 removed free features, [@olivierlambert](https://github.com/olivierlambert) ([Xen Orchestra](https://xen-orchestra.com) founder) decided to resurrect XCP, this time with a full community effort and not just a Citrix project.
 
-## Goals
+* [What is XCP-ng?](#what-is-xcp-ng-)
+  * [Genesis](#genesis)
+  * [Key points](#key-points)
+	
+* [Documentation](#documentation)
+* [Contribution & community](#contribution-and-community)
+* [Pro support](#pro-support)
 
-1. The main goal is to be able to enjoy Xenserver power (XAPI/features) with a real community backed solution (not "one company dependent")
-2. We want this to be 100% compatible with XenServer (as possible): ie being able to transfer VMs from XS to XCP-ng and vice-versa
-3. Same for XAPI to be 100% compatible with Open Source management solutions (Xen Orchestra)
+![](misc/assets/logo-glossy-small.png)
+
+## What is XCP-ng ?
+
+XCP-ng is a virtualization platform based on **Xen Source** and **Citrix® Hypervisor** (formerly XenServer).
+XCP-ng stands for **Xen Cloud Platform - New Generation** and is an *hommage* to the old Open Source project XCP that was ditched when XenServer became Open Sourced in 2013. 
```

Comment:
* "hommage" => "homage" or "tribute"?
* I'd avoid the word "ditched" which is slang. 
* "became Open Sourced" => "was open-sourced"?